.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "8"
+install:
+  - npm install
```

Comment:
Redundant, `npm install` is the default action for `node_js` language travis files.

Reply:
Removed